src/main/java/ch/njol/skript/expressions/ExprWhitelist.java

```diff
@@ -38,83 +39,86 @@
 import ch.njol.util.coll.CollectionUtils;
 
 @Name("Whitelist")
-@Description("A server's whitelist." +
-	"This expression can be used to add/remove players to/from the whitelist," +
-	" to enable it and disable it (set whitelist to true / set whitelist to false)," +
-	" and to empty it (reset whitelist)")
-@Examples({"set whitelist to false",
+@Description({
+	"The server's whitelist.",
+	"This expression can be used to add/remove players to/from the whitelist.",
+	"To enable and disable it (set whitelist to true/false), and to empty it (reset whitelist)"
+})
+@Examples({
+	"set whitelist to false",
 	"add all players to whitelist",
-	"reset the whitelist"})
+	"reset the whitelist"
+})
 @Since("2.5.2")
 public class ExprWhitelist extends SimpleExpression<OfflinePlayer> {
-	
+
 	static {
 		Skript.registerExpression(ExprWhitelist.class, OfflinePlayer.class, ExpressionType.SIMPLE, "[the] white[ ]list");
 	}
-	
+
 	@Override
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
 		return true;
 	}
-	
-	@Nullable
+
 	@Override
-	protected OfflinePlayer[] get(Event e) {
+	protected OfflinePlayer[] get(Event event) {
 		return Bukkit.getServer().getWhitelistedPlayers().toArray(new OfflinePlayer[0]);
 	}
-	
-	@Nullable
+
 	@Override
 	public Class<?>[] acceptChange(ChangeMode mode) {
-		if (mode == ChangeMode.ADD || mode == ChangeMode.REMOVE)
+		if (mode == ChangeMode.ADD || mode == ChangeMode.REMOVE) {
 			return CollectionUtils.array(OfflinePlayer[].class);
-		else if (mode == ChangeMode.SET || mode == ChangeMode.RESET)
+		} else if (mode == ChangeMode.SET || mode == ChangeMode.RESET) {
```

Comment:
DELETE changer should be added. It makes sense for `clear whitelist`

Also needs modification of the first condition in the change method when adding this.